src/Tickets/Commerce/Gateways/Stripe/Webhooks/Handler.php

```diff
@@ -65,9 +65,15 @@ public static function process_webhook_response( WP_REST_Request $request, WP_RE
 			return $events_map[ $type ]( $request, $response );
 		}
 
-		/**
-		 * @todo @moraleida We need to modify the order here based on the Payment Intent.
-		 */
+		// Define where this request should be processed and call that method
+		$handler = static::get_handler_method( $type );
+		$success = call_user_func( $handler, $event, $new_status, $request );
+
+		// Stripe webhooks don't care for anything other than our response codes
+		// 200 we're good. Anything else we're not.
+		if ( ! $success ) {
+			$response->set_status( 400 );
+		}
```

Comment:
by not passing the request and response we are limiting what we can do with the handler methods. So error handling will be required to happen here instead of the handler, which seems like an added layer.